COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -559,63 +556,39 @@ The COBOL Language Support extension checks for mistakes and errors in COBOL cod
-
-### Breadcrumb View
-
-The breadcrumb view across the top of the editor shows where the current line of code exists within the structure of the COBOL source code. You can click each element on the bar to see that section of the code highlighted, or to select a code element within the section and navigate to it.
-
-![](Images/code4z-img2.png)
-
-*Figure 2. Breadcrumb view.*
-
-### Outline View
-
-The Code4z package enables the outline view tab in the sidebar. Outline view enables you to easily navigate through sections and individual variables and paragraphs.
-
-### Shortcuts
-
-You can use the following shortcuts to navigate through your code:
-
-* **CTRL+ALT+,** moves you one section back.
-* **CTRL+ALT+.** moves you one section forward.
-* **CTRL+ALT+D** takes you to the data division.
-* **CTRL+ALT+P** takes you to the procedure division.
-* **CTRL+ALT+W** takes you to the working storage section.
-
-These commands are also available if you right click anywhere in the code.
+The COBOL Language Support extension enables several features for ease of navigation through code.
 
 ### Go To Definition
 
 While your cursor is placed on a variable or paragraph name, you can press **F12** or **CTRL+click** to use the **Go To Definition** functionality to display the point in the code where the variable or paragraph is defined.
 
 ![](Images/code4z-img3.png)
 
-*Figure 3. Go To Definition shows the point at which the USER-STREET variable is first defined.*
+*Figure 2. Go To Definition shows the point at which the USER-STREET variable is first defined.*
 
 ### Find All References
 
 The **Find All References** functionality (**SHIFT+ALT+F12**) highlights all references to a variable or paragraph and displays them in a list in the sidebar, so that you can easily navigate between them.
 
 ![](Images/code4z-img4.png)
 
-*Figure 4. Find All References lists all references to the USER-STREET variable in the code.*
+*Figure 3. Find All References lists all references to the USER-STREET variable in the code.*
 
 ## Copybook Support
 
-Copybooks are pieces of source code stored in separate data sets which are referenced in a program. The COBOL Language Support extension enables you to download all copybooks referenced in your program from the mainframe to a folder on your machine. In order for this feature to work, you need to set up and configure a Zowe CLI `zosmf` profile.
+Copybooks are pieces of source code stored in separate data sets which are referenced in a program. The COBOL Language Support extension enables you to download all copybooks referenced in your program from the mainframe to a folder in your workspace. In order for this feature to work, you need to set up and configure a Zowe CLI `zosmf` profile. You can also enable local copybook support, which enables copybook support features for copybooks stored locally in folders in your workspace, for example when accessing a COBOL project stored on a Github repository using VS Code.
```

Comment:
The last sentence is a little wordy. I recommend revising to:
You can also enable support for copybooks stored locally in folders in your workspace. This is useful when working with a COBOL project stored in a Github repository.